### PR TITLE
signup ui design fixed #118

### LINKED ISSUE #118: UI of register page
**Describe the new feature**
it can be more attractive

**Describe the possible approach**

**Describe any alternatives**

**Additional context**
please assign this to me under gssoc24 I will complete it very soon

### PR DESCRIPTION
**Pull request by riyasaxena32**
Screenshot (415)
Screenshot (416)
## Description

#118 issue fixed

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

usign signup button

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: client/src/pages/Signup/Signup.css
Comment (dvjsharma) [inline client/src/pages/Signup/Signup.css:1]: We don't use plain CSS. Please migrate to tailwind.

File: client/src/pages/Signup/Signup.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    color: white; /* White text color for the Login button */
+    padding: 10px 15px; /* Add some padding */
+    border-radius: 5px; /* Rounded corners */
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Slight shadow */
+  }
+  
+  .signup-card .text-center p a:hover {
+    background-color: #004d00; /* Darker green for the Login button on hover */
+    text-decoration: none; /* No underline on hover */
+  }
```
Comment (dvjsharma) [inline client/src/pages/Signup/Signup.css:84]: Please remove GPT comments :)

File: client/src/pages/Signup/signup.jsx [hunk 2]
```diff
@@ -3,293 +3,211 @@ import { Link } from "react-router-dom";
...
+        name.length > 0 &&
+        email.length > 0 &&
+        passwd.length > 0 &&
+        street.length > 0 &&
+        city.length > 0 &&
+        state.length > 0 &&
+        pincode.length > 0
+      )
+    ) {
+      toast.error("All fields are necessary");
```
Comment (dvjsharma) [inline client/src/pages/Signup/signup.jsx:33]: Please look for a better approach than checking the length. (empty functions etc.)

**Review by dvjsharma — CHANGES_REQUESTED** [review]
Major changes. PTAL.

**Comment by dvjsharma** [comment]
Good work with the design tho. Minor suggestion, please see if you can make the purple and blue backgrounds black and white so we can stick to the website theme.

File: client/src/pages/Signup/Signup.css
Comment (riyasaxena32) [reply client/src/pages/Signup/Signup.css:1]: i have used bootstrap

Comment (dvjsharma) [reply client/src/pages/Signup/Signup.css:1]: As I mentioned already, we use tailwind CSS. Please stick to that.

**Pull request closed without merging**